### DISCUSSION TITLE
State of the project

### DISCUSSION THREAD

**Discussion by ziomek64** [bloc97/Anime4K#241] [Q&A]
Is this project dead or is there not much to do?

**Comment by bloc97** [comment]
There's plans for a next version sometime in the future but right now I lack the time to continue working on this. Maybe if there's a breakthrough in neural network efficiency or consumer/gaming GPU processing power, we can make a next-gen version.

The major roadblocks right now are

1. Video ML software stack is not mature enough. If I use CUDA then there's no AMD/Intel GPU support, and HIP/SYCL support are not mature enough. You are very limited in the NN architectures you can make with GLSL shaders...
2. Consumer GPUs are too expensive, until more powerful GPUs become mainstream, even if I made a bigger network no one would be able to use it...
3. Lack of open anime datasets. All anime are copyrighted, and there's not enough data to lawfully pretrain a large-scale anime-related generative transformer. Having a model like that would lead to major breakthroughs in anime video upscaling.

**Reply by ybludo4ek** [reply]
Are we there yet? both amd and nvidia made decent gpus )
